Rest/Controller/ExportController.php

```diff
@@ -61,21 +81,142 @@ public function downloadAction($filePath, Request $request)
     }
 
     /**
-     * @param string $filePath
-     * @param \Symfony\Component\HttpFoundation\ServerBag $server
+     * @param $contentTypeIdList
+     * @param Request $request
+     *
+     * @return JsonResponse
+     *
+     * @throws ExportInProgressException
+     */
+    public function runExportAction($contentTypeIdList, Request $request)
+    {
+        $response = new JsonResponse();
+
+        if (!$this->authenticator->authenticate()) {
+            return $response->setStatusCode(401);
+        }
+
+        $options = $this->parseRequest($request);
+        $documentRoot = $options['documentRoot'];
+        unset($options['documentRoot']);
+
+        if ($this->fileSystem->isLocked()) {
+            $this->logger->warning('Export is running.');
+            throw new ExportInProgressException('Export is running');
+        }
+
+        $options['contentTypeIdList'] = $contentTypeIdList;
+
+        $optionString = '';
+        foreach ($options as $key => $option) {
+            $optionString .= !empty($option) ? ' --' . $key . '=' . $option : '';
+        }
+
+        $cmd = sprintf('%s/console ezreco:runexport %s --env=%s',
+            $this->kernelRootDir,
+            escapeshellcmd($optionString),
+            $this->kernelEnvironment
+        );
+
+        $command = sprintf(
```

Comment:
Please use `ProcessBuilder` as in this example: https://github.com/ezsystems/ezpublish-kernel/blob/master/eZ/Bundle/EzPublishCoreBundle/Command/ReindexCommand.php#L372